keps/sig-instrumentation/4785-crdmetrics-controller/kep.yaml

```diff
@@ -0,0 +1,38 @@
+title: CRDMetrics Controller
+kep-number: 4785
+authors:
+  - "@rexagod"
+owning-sig: sig-instrumentation
+participating-sigs:
+  - sig-instrumentation
+status: implementable
+creation-date: 2024-08-27
+reviewers:
+  - "@dgrisonnet"
+  - "@logicalhan"
+  - "@mrueg"
+approvers:
+  - "@dgrisonnet"
+  - "@logicalhan"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.32"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.32"
+
+# The following PRR answers are required at alpha release
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: N/A
```

Comment:
```suggestion
feature-gates: { } # no feature gates; this is an extension to Kubernetes
```

Reply:
Seems to fail with this change: https://prow.k8s.io/view/gs/kubernetes-ci-logs/pr-logs/pull/enhancements/4811/pull-enhancements-test/1854812490664251392, changed it to 

```
feature-gates:
  - name: ""
```

Reply:
Ah, sorry. `feature-gates: []` might work though.